app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        Person(person["name"], person["age"])
+    for el in people:
```

Comment:
Can you rename your variable "el"? Your variable is too short and isn't descriptive.
el -> human for example. 
You can check the checklist for details.